src/ring_theory/dedekind_domain.lean

```diff
@@ -108,33 +108,150 @@ structure is_dedekind_domain_dvr : Prop :=
 (is_dvr_at_nonzero_prime : ∀ P ≠ (⊥ : ideal A), P.is_prime →
   discrete_valuation_ring (localization.at_prime P))
 
+section inverse
+
+open_locale classical
+
+variables {R₁ : Type*} [integral_domain R₁] {g : fraction_map R₁ K}
+
+noncomputable instance fractional_ideal_has_div :
```

Comment:
If we put a `group_with_zero` instance on `fraction_map` (for dedekind domains), then we will have two `has_div` instances. That's unfortunate...
I don't see an obvious solution.